dotcom-rendering/src/components/FootballTable.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import type { Meta, StoryObj } from '@storybook/react/*';
+import { allModes } from '../../.storybook/modes';
+import { FootballTable } from './FootballTable';
+
+const meta = {
+	title: 'Components/Football Table',
+	component: FootballTable,
+	decorators: [
+		// To make it easier to see the top border above the table
+		(Story) => (
+			<>
+				<div css={{ padding: 4 }}></div>
+				<Story />
+			</>
+		),
+	],
+	parameters: {
+		chromatic: {
+			modes: {
+				'vertical mobile': allModes['vertical mobile'],
+				'vertical desktop': allModes['vertical desktop'],
+				'vertical wide': allModes['splitVertical'],
+			},
+		},
+	},
+} satisfies Meta<typeof FootballTable>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
```

Comment:
Given this only has one story, you can give it the same name as the component so that it gets hoisted[^1] in the UI:

```suggestion
import { FootballTable as FootballTableComponent } from './FootballTable';

const meta = {
	title: 'Components/Football Table',
	component: FootballTableComponent,
	decorators: [
		// To make it easier to see the top border above the table
		(Story) => (
			<>
				<div css={{ padding: 4 }}></div>
				<Story />
			</>
		),
	],
	parameters: {
		chromatic: {
			modes: {
				'vertical mobile': allModes['vertical mobile'],
				'vertical desktop': allModes['vertical desktop'],
				'vertical wide': allModes['splitVertical'],
			},
		},
	},
} satisfies Meta<typeof FootballTableComponent >;

export default meta;

type Story = StoryObj<typeof meta>;

export const FootballTable = {
```

[^1]: https://storybook.js.org/docs/writing-stories/naming-components-and-hierarchy#single-story-hoisting